bin/crystalball

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
```

Comment:
I wonder if it does anything not being the first line

Reply:
it does